src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -282,6 +282,15 @@ const generateTaskRole = (taskRole, labels, config) => {
     gangAllocation = 'false';
     retryPolicy.fancyRetryPolicy = true;
   }
+  // calculate pod priority
+  // reference: https://github.com/microsoft/pai/issues/3704
+  let jobPriority = 127;
```

Comment:
>jobPriority [](start = 6, length = 11)

Can we enforce hivedPodSpec.priority in validation [-1, 126]

Reply:
it shouldn't be validated here, can be done in hived spec validation.
but at present the priority class uses hard coded values and there's no such validation for the whole hived spec yet.

Reply:
Ok, so for safety can we round overflowed jobPriority to be 126, and round undherflowed jobPriority to be -1?

Reply:
I mean orignal jobPriority to within  [-1, 126], i.e. round hivedPodSpec.priority within it :)